test/base/test_readwrite.jl

```diff
@@ -1,4 +1,6 @@
-using ITensors, HDF5, Test
+using HDF5
+using ITensors
+using Test
```

Comment:
Would you mind updating the style of this test, i.e. wrapping in a module and changing to explicit using statements?

Reply:
No problem